src/mono/wasm/build/WasmApp.targets

```diff
@@ -185,6 +189,16 @@
      Assemblies="@(_WasmPInvokeAssemblies)"
      OutputPath="$(WasmBuildDir)pinvoke-table.h" />
 
+   <!-- ICall table generation -->
+   <Exec Condition="'$(WasmLinkIcalls)' == 'true'" Command="$(_CrossCompilerPath) --print-icall-table > $(WasmBuildDir)/runtime-icall-table.h" />
```

Comment:
we're mixing forward and backward slashes in the paths here and above, do you know whether that'll work right on windows and mac?

Reply:
Not sure, we can check when we try to run on windows.